metals/src/main/scala/scala/meta/internal/metals/codeactions/ImportMissingSymbol.scala

```diff
@@ -40,8 +38,9 @@ class ImportMissingSymbol(compilers: Compilers, buildTargets: BuildTargets)
       codeAction
         .getEdit()
         .getChanges()
-        .getOrDefault(uri, Collections.emptyList)
```

Comment:
This is another one :/

Changes would have `file:///C:/Users/Tomasz/Documents/Work/CodeActionImportIssue/subprojecta/src/main/scala/SomeClassA.scala`

while uri was `"file:///c%3A/Users/Tomasz/Documents/Work/CodeActionImportIssue/subprojecta/src/main/scala/SomeClassA.scala`